File: Patterns/pattern5.java
package Patterns;
//inverted half pyramid (rotated by 180 degrees)



public class pattern5 {
    public static void main(String[] args) {
        int n = 4;
        for(int i = 1;i<=n;i++) //no of rows (outer loop)
        {
            //inner loop1(SPACE)
            for(int j=1;j<=(n-i);j++){
               System.out.print("  ");
            }
            //inner loop2(STAR)
            for(int j=1;j<=i;j++){
                System.out.print("* ");
               }
        System.out.println();
        }
    }
}

File: DSA/Patterns/pattern5.cpp
/*
*****
****
***
**
*
*/

#include <iostream>
using namespace std;

void pattern5(){
    for (int i=1;i<=5;i++){
        for(int j=1;j<=(5-(i-1));j++){
            cout<<"*";
        }
        cout<<endl;
    }

    //OR:

    // for (int i = 1; i<=5;i++){
    //     for(int j=5;j>=i;j--){
    //         cout<<"*";
    //     }
    //     cout<<endl;
    // }
}

int main(){
    pattern5();
    return 0;
}